demo/nbody/src/main.rs

```diff
@@ -1,111 +1,123 @@
+ #![feature(augmented_assignments)]
```

Comment:
Only thing is the `*=` operators from cgmath. I could just use regular re-assignments, which I'm guessing should optimize ok, which mean we could drop cgmath's `"unstable"` feature flag.